### DISCUSSION TITLE
Xournal on HP Spectre 2019 Running windows 10, touch detection and pen button pressing

### DISCUSSION THREAD

**Discussion by Jediparth** [xournalpp/xournalpp#3102] [Q&A]
Hello,
I'm a college student in need of a PDF editing app and I found Xournal++ to be the perfect solution for me, however, I was annoyed at the fact that whenever I touched the screen with my hand (and not my stylus) it would begin drawing. I wished that there was an option to disable and enable hand drawing like there is with one-note for windows 10. 

I have attempted to search for solutions online, and all I have found are patched for the original Xournal, which I have no idea how to install and work with, especially since all the tutorials describing such are for Linux and/or Mac systems.

I also am having trouble getting the program to recognize my stylus buttons on my HP Spectre 2019 stylus, despite trying the settings.

Can anybody direct me to a plug-in that would perhaps give support for my stylus and/or a plugin that can disable touch on Xournal++, or direct me on how to do so with even the original version of Xournal on windows? 

Thanks in advance.

**Comment by rolandlo** [comment]
What version of Xournal++ are you using? Expect best results with [version 1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly). 
There are various settings in the Touchscreen panel in menu `Edit > Preferences` related to touch detection. Have you tried any of these? In particular you can enable and disable Touch Drawing there.
Pen button pressing under Windows has some problems. See and #2894 and #2316 for more info and workarounds.

**Reply by Jediparth** [reply]
Yes, I have tried the anti touch detection in preferences, however it only disables touch detection for around 10 seconds maximum. I would like to be able to fully disable it and use my hand to scroll, much like is done in OneNote. I saw a patch for that for the old Xournal (Not Xournal++) on source forge but I’m mot sure if it’s even compatible, and second, I’m just jot sure how I would go about installing that.

Maybe there’s some kind of plugin?
